weed/filer/redis2/universal_redis_store_test.go

```
@@ -0,0 +1,268 @@
+	if os.Getenv("RUN_REDIS_TESTS") != "1" {
+		t.Skip("redis2 tests are disabled. Start a redis-server and set RUN_REDIS_TESTS=1 to enable, REDIS_ADDR defaults to 127.0.0.1:6379.")
+	}
+
+	addr := os.Getenv("REDIS_ADDR")
+	if addr == "" {
+		addr = "127.0.0.1:6379"
+	}
+
+	ctx := context.Background()
+	client := redis.NewClient(&redis.Options{Addr: addr})
+	t.Cleanup(func() { client.Close() })
```

golangci-lint (2.12.2)

[error] Error return value of `client.Close` is not checked

(errcheck)